terminus-ui/cohort-date-range/src/cohort-date-range.component.md

```diff
@@ -0,0 +1,95 @@
+<h1>Cohort Date Range</h1>
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Basic usage](#basic-usage)
+- [Event driven](#event-driven)
+- [Inputs](#inputs)
+  - [allowCustomDates](#allowCustomDates)
+  - [Disable the component](#disable-the-component)
```

Comment:
This ToC isn't formatted like the others; it looks like it was manually edited vs generated?